deploy/operator/clean-shared-host.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -o verbose
+set -eu
```

Comment:
Let's add also the `set -o pipefail` to catch when a command in a piped chain fails